klpbuild/klplib/patch.py

```diff
@@ -32,20 +32,21 @@ def analyse_files(cs_list, sle_patches):
             bc = cs.base_cs_name()
             patches = sle_patches[bc]
             branch = KERNEL_BRANCHES[bc]
-            files = get_patch_files(patches, branch)
-            fconfigs, _, missing = find_configs_for_files(cs, files)
+            files_funcs = get_patches_files(patches, branch)
+            files, _ = find_configs_for_files(cs, files_funcs.keys())
```

Comment:
```suggestion
            files_config, _ = find_configs_for_files(cs, files_funcs.keys())
```
What about renaming this? Even with another name that makes it clear that the dict contains the config and object.

Reply:
done